server/requirements.txt

```diff
@@ -41,4 +41,4 @@ wooper==0.4.1
 git+git://github.com/superdesk/eve@upstream-nk#egg=Eve==0.6-dev
 git+git://github.com/amagdas/tinys3@list-buckets#egg=tinys3
 -e hg+http://bitbucket.org/haypo/hachoir3/#egg=hachoir3
-git+git://github.com/liveblog/superdesk.git@1eab0207a09f026c7875239da94a1d9a3199b14a#egg=Superdesk-Server
+git+git://github.com/liveblog/superdesk#egg=Superdesk-Server
```

Comment:
It's important to keep a reference of the version here to keep the tests consistent.
@actionless gave me a lesson already :trollface: